pkg/controller/pod_driver.go

```diff
@@ -130,13 +130,20 @@ func (p *PodDriver) checkAnnotations(ctx context.Context, pod *corev1.Pod) error
 	lock.Lock()
 	defer lock.Unlock()
 
+	// Add a 10s delay to avoid misjudgment of deletion due to cache reasons when the pod is just created
```

Comment:
we're not sure of the root cause yet, AFAIK if no `ResourceVersion="0"` is set, the apiserver always query etcd to get the latest state. 

```suggestion
	// Add a 10s delay to avoid misjudgment of deletion due to rare ApiServer info mismatch when the pod is just created
```